pipeline/tasks/format_tasks.py

```diff
@@ -65,35 +65,62 @@ def validate(self):
 
 
 class GoFormatTask(task_base.TaskBase):
-    def execute(self, intermediate_code_dir):
-        print 'Formatting files in ' + os.path.abspath(intermediate_code_dir)
-        self.exec_command(['gofmt', '-w', intermediate_code_dir])
+    def execute(self, gapic_code_dir):
+        print 'Formatting files in ' + os.path.abspath(gapic_code_dir)
+        self.exec_command(['gofmt', '-w', gapic_code_dir])
 
     def validate(self):
         return [go_requirements.GoFormatRequirements]
 
 
 class RubyFormatTask(task_base.TaskBase):
-    def execute(self, intermediate_code_dir):
-        print 'Formatting file in ' + os.path.abspath(intermediate_code_dir)
-        subprocess.call(['rubocop', '-a', intermediate_code_dir])
+    def execute(self, gapic_code_dir):
+        print 'Formatting file in ' + os.path.abspath(gapic_code_dir)
+        subprocess.call(['rubocop', '-a', gapic_code_dir])
 
     def validate(self):
         return [ruby_requirements.RubyFormatRequirements]
 
 
 class PhpFormatTask(task_base.TaskBase):
-    def execute(self, intermediate_code_dir):
-        abs_code_dir = os.path.abspath(intermediate_code_dir)
+    def execute(self, gapic_code_dir):
+        abs_code_dir = os.path.abspath(gapic_code_dir)
         print 'Formatting file using php-cs-fixer in ' + abs_code_dir
-        subprocess.call(['php-cs-fixer', 'fix', intermediate_code_dir])
+        subprocess.call(['php-cs-fixer', 'fix', gapic_code_dir])
         # We require a second call to php-cs-fixer because instances of @type
         # have been converted to @var. We cannot disable this conversion in
         # the first call without affecting other aspects of the formatting.
-        subprocess.call(['php-cs-fixer', 'fix', intermediate_code_dir,
+        subprocess.call(['php-cs-fixer', 'fix', gapic_code_dir,
                          '--fixers=phpdoc_var_to_type'])
         print 'Formatting file using phpcbf in ' + abs_code_dir
-        subprocess.call(['phpcbf', '--standard=PSR2', intermediate_code_dir])
+        subprocess.call(['phpcbf', '--standard=PSR2', gapic_code_dir])
 
     def validate(self):
         return [php_requirements.PhpFormatRequirements]
+
+
+class CSharpFormatTask(task_base.EmptyTask):
+    pass
+
+
+class NodeJSFormatTask(task_base.EmptyTask):
+    pass
+
+
+format_task_dict = {
+    'java': JavaFormatTask,
+    'python': PythonFormatTask,
+    'go': GoFormatTask,
+    'ruby': RubyFormatTask,
```

Comment:
In the meanwhile I've removed RubyFormatTask. Please replace this by an EmptyTask